diagnostic.md

```diff
@@ -93,5 +98,6 @@ var teamMember = {
 
 What is the value of `teamMember['hobby']`?  Of `teamMember['name']`?  Of
 `teamMember['name']['surname']`?
-
-_Replace this text with your answer._
+1) exercise
+2) Jason Weeks
+3) Jason Weeks, Weeks
```

Comment:
teamMember['name']['surname'] is specifically pointing to the value of 'surname' in 'name', which would just be 'Weeks'.